minecraft-server/start-configuration

```diff
@@ -73,6 +73,10 @@ case "$TYPE" in
     exec /start-deployVanilla $@
   ;;
 
+  SPONGEVANILLA|spongevanilla)
```

Comment:
Since I haven't used Sponge personally, would it be confusing to shorten this to `SPONGE` or `sponge`?

Reply:
Yeah. SPONGEFORGE also exists and is the next task in my list.